lib/src/marked-options.ts

```diff
@@ -58,19 +33,4 @@ export class MarkedOptions {
    * Use smarter list behavior than the original markdown. May eventually be default with the old behavior moved into pedantic.
    */
   smartLists?: boolean;
```

Comment:
`smartlists` was also removed a while ago, before the other options.

Reply:
oh I thought it was something specific for `ngx-markdown` because I didn't see at all in marked docs
removed it as well now